src/util.ts

```diff
@@ -54,3 +56,28 @@ export function stringsOnly(env: { [envVar: string]: string | null | undefined }
 	}
 	return result;
 }
+
+export function normalizePathOrFileUrlToPath(p: string): string {
+	if (p?.startsWith("file://")) {
+		p = url.fileURLToPath(p);
+	}
+	return normalizePath(p);
+}
+
+export function normalizePath(p: string): string {
+	if (!p) {
+		return p;
+	}
+	if (process.platform === 'win32') {
+		// On Windows, normalize drive letter to upper case. Works around
+		// https://github.com/microsoft/vscode/issues/68325. A mismatch in
```

Comment:
I regret having to do this. I'm working behind the scenes to try to get VS Code to take another look at addressing this for everyone.

Reply:
Linking to https://github.com/microsoft/vscode/issues/68325